values.yaml

```diff
@@ -124,3 +124,10 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+pvc:
+  enabled: true
+  storageClassName: managed-premium
```

Comment:
Why premium and not default @michalziobro ?
https://docs.microsoft.com/en-us/azure/aks/concepts-storage

Reply:
yes, could be default here